lib/app.rb

```diff
@@ -0,0 +1,71 @@
+require_relative "./item_repository"
+require_relative "./order_repository"
+require_relative "./database_connection"
+
+class Application
+  def initialize(database_name, io, item_repository, order_repository)
+    DatabaseConnection.connect(database_name)
+    @io = io
+    @item_repository = item_repository
+    @order_repository = order_repository
+  end
+
```

Comment:
This method is very long. I think it would be better to break up some of its functionality into separate methods.